.github/workflows/ci-test-ginkgo.yml

```
@@ -18,14 +18,14 @@ on:
 
 jobs:
   build:
-    name: Auto-testing Framework / ${{ matrix.os }} / ${{ matrix.runtime }}
-    runs-on: ${{ matrix.os }}
+    name: Auto-testing Framework / ${{ matrix.labels }} / ${{ matrix.runtime }}
+    runs-on: ${{ matrix.labels }}
```

```suggestion
    name: Auto-testing Framework / ${{ matrix.runtime }}
    runs-on:
        labels: ubuntu-latest-8-cores
```

We don't need a matrix, if it's only going to be a single runner. And since all the runners are anyway coming with BTF there's no point in testing across versions since the matrix was introduced to test across kernel versions.

---

sure, removed the matrix for runs-on.